generate_html.py

```diff
@@ -23,11 +23,16 @@ def make_threads_link(val):
     # target _blank to open new window
     return '<a target="_blank" href="https://www.threads.net/@{}">{}</a>'.format(val[1], val[0])
 
+def make_twitter_link(val):
```

Comment:
I left out Twitter links on purpose. I don't want to drive any traffic there. This is meant to be a resource for people to look up accounts they know from Twitter to find them on Threads.

Reply:
roger that